pkg/crc/machine/machine.go

```diff
@@ -110,12 +144,18 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 		result.Status = vmState.String()
 	} else {
-		host, err := libMachineAPIClient.Load(machineConfig.Name)
+		host, err := libMachineAPIClient.Load(startConfig.Name)
 		if err != nil {
 			result.Error = err.Error()
 			return *result, errors.Newf("Error loading host: %v", err)
 		}
 
+		bundleName, _ := host.Driver.GetBundleName()
+		crcBundleMetadata, err = loadCrcBundleInfo(&startConfig, bundleName)
+		if err != nil {
+			result.Error = err.Error()
+			return *result, err
```

Comment:
`err` should be wrapped with `errors.Newf` as we have every place when returning back to cmd side.